umi_tools/umi_methods.py

```diff
@@ -370,7 +386,7 @@ def fill(self):
             if read.is_read2:
                 continue
 
-            self.umis[get_umi(read)] += 1
+            self.umis[get_umi(read, umi_sep, umi_tag, get_umi_method)] += 1
```

Comment:
Previously we were using the get_umi function without passing the user defined umi separator! This is an bug I introduced when I added the --umi-separator option.